packages/react-data-grid/src/Row.tsx

```diff
@@ -62,7 +62,7 @@ export default class Row<R extends {}> extends React.Component<RowRendererProps<
       lastFrozenColumnIndex
     };
 
-    return <Renderer key={`${key}-${idx}`} {...cellProps} />;
+    return <Renderer key={`${key as keyof R}-${idx}`} {...cellProps} />; // FIXEME: fix key type
```

Comment:
```suggestion
    return <Renderer key={`${key as keyof R}-${idx}`} {...cellProps} />; // FIXME: fix key type
```
Might miss this one when searching for `fixme`.